content/ja/examples/service/access/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM nginx:1.9.14
+FROM nginx:1.17.3
```

Comment:
@sftim IIRC, we just change the en content and let localization team change the related localized content, right?